raft/raft.cpp

```diff
@@ -1512,6 +1512,12 @@ raft::add_self_to_swarm()
     const auto this_address = std::find_if(all_peers.begin(), all_peers.end(),
             [&](const auto& peer){return peer.uuid == this->get_uuid();});
 
+    if (this_address == all_peers.end())
+    {
+        LOG(error) << "Unable to add this peer to the swarm. It may not be in the peers.json file, or it may be blacklisted.";
+        return;
```

Comment:
In this case we couldn't add ourselves to the swarm. Is that fatal? If so show we throw here?

Reply:
Yeah, I was struggling with that. It could simply be that the owner of the node did not add the node info to peers.json... I thought that that wasn't as bad a thing as a blacklisted peer..

But, now that I think about it, bailing right away seems like the nicest thing to do...

Reply:
Now the blacklist test is done in raft, in the case that the node listing was not in the default quorum.